clients/pkg/logentry/stages/pipeline.go

```diff
@@ -68,6 +72,19 @@ func RunWith(input chan Entry, process func(e Entry) Entry) chan Entry {
 
 // Run implements Stage
 func (p *Pipeline) Run(in chan Entry) chan Entry {
+	if InspectWithId {
+		// add a first stage to add the inspect id
+		in = RunWith(in, func (e Entry) Entry {
+			var inspectId [16]byte
+
+			rand.Read(inspectId[:])
+
+			e.Extracted["InspectId"] = hex.EncodeToString(inspectId[:])
```

Comment:
I'm wondering now if we might kill two birds with one stone if we `md5`-d the log line. That'd both provide a way to parse the interleaved output _and_ isolate the offending log in the output. What do you think?

Reply:
I think this would actually be the correct approach, if you are fine with adding an ID to the Entry struct, then I can remove all of that

Reply:
Sounds good :+1: